superset-frontend/src/cccs-viz/plugins/plugin-chart-cccs-grid/src/components/controls/JumpActionConfigControll/index.tsx

```diff
@@ -0,0 +1,207 @@
+/**
```

Comment:
Nitpick in the name, there`s a typo, should be JumpActionConfigControl, there`s one too many l at the end of the name